.grit/patterns/java/no_big_decimal_double.md

```diff
@@ -1,10 +1,8 @@
 ---
-title: "BigDecimal(double)" should not be used
+title: "`BigDecimal(double)` should not be used"
 tags: [java]
 ---
 
-# "BigDecimal(double)" should not be used
```

Comment:
I feel like using `h1` as the title is more self-descriptive for patterns as opposed to relying on the `title` frontmatter.

In general, I think a good litmus test here is for the frontmatter to be completely optional by default, and I think this change moves in the opposite direction.

Thoughts?

Reply:
Both are allowed - frontmatter is optional, and you can just use h1. In this case we had both so I removed one.

Many patterns still use an H1.